examples/gallery/symbols/points_categorical.py

```diff
@@ -36,8 +36,8 @@
     region=region,
     projection="X10c/10c",
     frame=[
-        'xafg+l"Bill length (mm)"',
-        'yafg+l"Bill depth (mm)"',
+        'xafg+l"Bill length in mm"',
+        'yafg+l"Bill depth in mm"',
```

Comment:
Is this documented in "NIST unit rules and style conventions"? Personally, I prefer "Bill length (mm)" and I think most publications use this style in their figure labels.

Reply:
> Is this documented in "NIST unit rules and style conventions"? Personally, I prefer "Bill length (mm)" and I think most publications use this style in their figure labels.

I also prefer "Bill length (mm)".

Reply:
I double-checked the conventions and "Bill length (mm)" is a correct way to make the annotation (see **7.1.** under https://www.nist.gov/pml/special-publication-811/nist-guide-si-chapter-7-rules-and-style-conventions-expressing-values). Will adjust it.